From 80d1e37696ef362222d233ec3fb9938f5235a396 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Thu, 6 Aug 2020 16:32:31 +0800
Subject: [PATCH] ARM: dts: rockchip: rename rv11xx to rv1126

rv11xx-evb-v13.dtsi -> rv1126-evb-v13.dtsi
rv11xx-evb-v12.dtsi -> rv1126-evb-v12.dtsi
rv11xx-evb-v10.dtsi -> rv1126-evb-v10.dtsi
rv11xx-evb-v10-robot.dtsi -> rv1126-evb-v10-robot.dtsi
rv11xx-thunder-boot-emmc.dtsi -> rv1126-thunder-boot-emmc.dtsi
rv11xx-thunder-boot-spi-nor.dtsi -> rv1126-thunder-boot-spi-nor.dtsi
rv11xx-thunder-boot.dtsi -> rv1126-thunder-boot.dtsi
rv11xx-ai-cam.dtsi -> rv1126-ai-cam.dtsi
rv11xx-evb-thunder-boot.dtsi -> rv1126-evb-thunder-boot.dtsi
rv11xx-ipc.dtsi -> rv1126-ipc.dtsi

Change-Id: I815b0d1b471c9b14d627cff7fad961290c7f043f
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 arch/arm/boot/dts/rv1109-38-v10-spi-nand.dts                | 2 +-
 arch/arm/boot/dts/rv1109-ai-cam-ddr3-v1.dts                 | 2 +-
 arch/arm/boot/dts/rv1109-evb-ddr3-v12-facial-gate.dts       | 2 +-
 arch/arm/boot/dts/rv1126-38x38-v10-spi-nor.dts              | 2 +-
 arch/arm/boot/dts/rv1126-ai-cam-ddr3-v1.dts                 | 2 +-
 .../arm/boot/dts/{rv11xx-ai-cam.dtsi => rv1126-ai-cam.dtsi} | 0
 arch/arm/boot/dts/rv1126-bat-ipc-v10.dts                    | 2 +-
 arch/arm/boot/dts/rv1126-evb-ddr3-v10-robot.dts             | 4 ++--
 arch/arm/boot/dts/rv1126-evb-ddr3-v10.dts                   | 2 +-
 arch/arm/boot/dts/rv1126-evb-ddr3-v12-robot.dts             | 4 ++--
 arch/arm/boot/dts/rv1126-evb-ddr3-v12-spi-nor.dts           | 2 +-
 arch/arm/boot/dts/rv1126-evb-ddr3-v12-tb-emmc.dts           | 6 +++---
 arch/arm/boot/dts/rv1126-evb-ddr3-v12-tb-spi-nor.dts        | 6 +++---
 arch/arm/boot/dts/rv1126-evb-ddr3-v12.dts                   | 2 +-
 arch/arm/boot/dts/rv1126-evb-ddr3-v13-robot.dts             | 4 ++--
 arch/arm/boot/dts/rv1126-evb-ddr3-v13-tb-emmc.dts           | 6 +++---
 arch/arm/boot/dts/rv1126-evb-ddr3-v13.dts                   | 2 +-
 ...x-evb-thunder-boot.dtsi => rv1126-evb-thunder-boot.dtsi} | 0
 ...{rv11xx-evb-v10-robot.dtsi => rv1126-evb-v10-robot.dtsi} | 0
 .../boot/dts/{rv11xx-evb-v10.dtsi => rv1126-evb-v10.dtsi}   | 0
 .../boot/dts/{rv11xx-evb-v12.dtsi => rv1126-evb-v12.dtsi}   | 2 +-
 .../boot/dts/{rv11xx-evb-v13.dtsi => rv1126-evb-v13.dtsi}   | 2 +-
 arch/arm/boot/dts/{rv11xx-ipc.dtsi => rv1126-ipc.dtsi}      | 0
 arch/arm/boot/dts/rv1126-ipc2-ddr3-v10.dts                  | 2 +-
 arch/arm/boot/dts/rv1126-sphericalipc-ddr3-v10.dts          | 2 +-
 ...thunder-boot-emmc.dtsi => rv1126-thunder-boot-emmc.dtsi} | 2 +-
 ...r-boot-spi-nor.dtsi => rv1126-thunder-boot-spi-nor.dtsi} | 2 +-
 .../{rv11xx-thunder-boot.dtsi => rv1126-thunder-boot.dtsi}  | 0
 28 files changed, 31 insertions(+), 31 deletions(-)
 rename arch/arm/boot/dts/{rv11xx-ai-cam.dtsi => rv1126-ai-cam.dtsi} (100%)
 rename arch/arm/boot/dts/{rv11xx-evb-thunder-boot.dtsi => rv1126-evb-thunder-boot.dtsi} (100%)
 rename arch/arm/boot/dts/{rv11xx-evb-v10-robot.dtsi => rv1126-evb-v10-robot.dtsi} (100%)
 rename arch/arm/boot/dts/{rv11xx-evb-v10.dtsi => rv1126-evb-v10.dtsi} (100%)
 rename arch/arm/boot/dts/{rv11xx-evb-v12.dtsi => rv1126-evb-v12.dtsi} (97%)
 rename arch/arm/boot/dts/{rv11xx-evb-v13.dtsi => rv1126-evb-v13.dtsi} (89%)
 rename arch/arm/boot/dts/{rv11xx-ipc.dtsi => rv1126-ipc.dtsi} (100%)
 rename arch/arm/boot/dts/{rv11xx-thunder-boot-emmc.dtsi => rv1126-thunder-boot-emmc.dtsi} (94%)
 rename arch/arm/boot/dts/{rv11xx-thunder-boot-spi-nor.dtsi => rv1126-thunder-boot-spi-nor.dtsi} (89%)
 rename arch/arm/boot/dts/{rv11xx-thunder-boot.dtsi => rv1126-thunder-boot.dtsi} (100%)

diff --git a/arch/arm/boot/dts/rv1109-38-v10-spi-nand.dts b/arch/arm/boot/dts/rv1109-38-v10-spi-nand.dts
index 7444ebc21432..db7f9ad85ddb 100644
--- a/arch/arm/boot/dts/rv1109-38-v10-spi-nand.dts
+++ b/arch/arm/boot/dts/rv1109-38-v10-spi-nand.dts
@@ -5,7 +5,7 @@
 
 /dts-v1/;
 #include "rv1109.dtsi"
-#include "rv11xx-ipc.dtsi"
+#include "rv1126-ipc.dtsi"
 
 / {
 	model = "Rockchip RV1109 38 V10 SPI NAND DDR3 Board";
diff --git a/arch/arm/boot/dts/rv1109-ai-cam-ddr3-v1.dts b/arch/arm/boot/dts/rv1109-ai-cam-ddr3-v1.dts
index ef20a43f2908..aee2210f9c80 100644
--- a/arch/arm/boot/dts/rv1109-ai-cam-ddr3-v1.dts
+++ b/arch/arm/boot/dts/rv1109-ai-cam-ddr3-v1.dts
@@ -5,7 +5,7 @@
 
 /dts-v1/;
 #include "rv1109.dtsi"
-#include "rv11xx-ai-cam.dtsi"
+#include "rv1126-ai-cam.dtsi"
 
 / {
 	model = "Rockchip RV1109 AI CAMERA DDR3L Board";
diff --git a/arch/arm/boot/dts/rv1109-evb-ddr3-v12-facial-gate.dts b/arch/arm/boot/dts/rv1109-evb-ddr3-v12-facial-gate.dts
index 4e366acf4476..7b9337c988f7 100644
--- a/arch/arm/boot/dts/rv1109-evb-ddr3-v12-facial-gate.dts
+++ b/arch/arm/boot/dts/rv1109-evb-ddr3-v12-facial-gate.dts
@@ -4,7 +4,7 @@
  */
 /dts-v1/;
 #include "rv1109.dtsi"
-#include "rv11xx-evb-v12.dtsi"
+#include "rv1126-evb-v12.dtsi"
 / {
 	model = "Rockchip RV1109 V12 FACIAL GATE Board";
 	compatible = "rockchip,rv1109-evb-ddr3-v12-facial-gate", "rockchip,rv1109";
diff --git a/arch/arm/boot/dts/rv1126-38x38-v10-spi-nor.dts b/arch/arm/boot/dts/rv1126-38x38-v10-spi-nor.dts
index d2634eb0be4c..498d1425d1d5 100644
--- a/arch/arm/boot/dts/rv1126-38x38-v10-spi-nor.dts
+++ b/arch/arm/boot/dts/rv1126-38x38-v10-spi-nor.dts
@@ -5,7 +5,7 @@
 
 /dts-v1/;
 #include "rv1126.dtsi"
-#include "rv11xx-ipc.dtsi"
+#include "rv1126-ipc.dtsi"
 
 / {
 	model = "Rockchip RV1126 38x38 V10 SPI NOR DDR3 Board";
diff --git a/arch/arm/boot/dts/rv1126-ai-cam-ddr3-v1.dts b/arch/arm/boot/dts/rv1126-ai-cam-ddr3-v1.dts
index 16ee0fa265c1..23050d251cf7 100644
--- a/arch/arm/boot/dts/rv1126-ai-cam-ddr3-v1.dts
+++ b/arch/arm/boot/dts/rv1126-ai-cam-ddr3-v1.dts
@@ -5,7 +5,7 @@
 
 /dts-v1/;
 #include "rv1126.dtsi"
-#include "rv11xx-ai-cam.dtsi"
+#include "rv1126-ai-cam.dtsi"
 
 / {
 	model = "Rockchip RV1126 AI CAMERA DDR3L Board";
diff --git a/arch/arm/boot/dts/rv11xx-ai-cam.dtsi b/arch/arm/boot/dts/rv1126-ai-cam.dtsi
similarity index 100%
rename from arch/arm/boot/dts/rv11xx-ai-cam.dtsi
rename to arch/arm/boot/dts/rv1126-ai-cam.dtsi
diff --git a/arch/arm/boot/dts/rv1126-bat-ipc-v10.dts b/arch/arm/boot/dts/rv1126-bat-ipc-v10.dts
index ebef28b063af..26be3e5737a0 100644
--- a/arch/arm/boot/dts/rv1126-bat-ipc-v10.dts
+++ b/arch/arm/boot/dts/rv1126-bat-ipc-v10.dts
@@ -5,7 +5,7 @@
 
 /dts-v1/;
 #include "rv1126.dtsi"
-#include "rv11xx-ipc.dtsi"
+#include "rv1126-ipc.dtsi"
 #include <dt-bindings/input/input.h>
 
 / {
diff --git a/arch/arm/boot/dts/rv1126-evb-ddr3-v10-robot.dts b/arch/arm/boot/dts/rv1126-evb-ddr3-v10-robot.dts
index 7cf16e1cd8de..233d480cea88 100644
--- a/arch/arm/boot/dts/rv1126-evb-ddr3-v10-robot.dts
+++ b/arch/arm/boot/dts/rv1126-evb-ddr3-v10-robot.dts
@@ -5,8 +5,8 @@
 
 /dts-v1/;
 #include "rv1126.dtsi"
-#include "rv11xx-evb-v10.dtsi"
-#include "rv11xx-evb-v10-robot.dtsi"
+#include "rv1126-evb-v10.dtsi"
+#include "rv1126-evb-v10-robot.dtsi"
 
 / {
 	model = "Rockchip RV1126 EVB DDR3 Board For Robot";
diff --git a/arch/arm/boot/dts/rv1126-evb-ddr3-v10.dts b/arch/arm/boot/dts/rv1126-evb-ddr3-v10.dts
index 34d06abb1bea..17ab37268020 100644
--- a/arch/arm/boot/dts/rv1126-evb-ddr3-v10.dts
+++ b/arch/arm/boot/dts/rv1126-evb-ddr3-v10.dts
@@ -5,7 +5,7 @@
 
 /dts-v1/;
 #include "rv1126.dtsi"
-#include "rv11xx-evb-v10.dtsi"
+#include "rv1126-evb-v10.dtsi"
 
 / {
 	model = "Rockchip RV1126 EVB DDR3 V10 Board";
diff --git a/arch/arm/boot/dts/rv1126-evb-ddr3-v12-robot.dts b/arch/arm/boot/dts/rv1126-evb-ddr3-v12-robot.dts
index cb88fe43da89..16c1086e6420 100644
--- a/arch/arm/boot/dts/rv1126-evb-ddr3-v12-robot.dts
+++ b/arch/arm/boot/dts/rv1126-evb-ddr3-v12-robot.dts
@@ -4,8 +4,8 @@
  */
 /dts-v1/;
 #include "rv1126.dtsi"
-#include "rv11xx-evb-v12.dtsi"
-#include "rv11xx-evb-v10-robot.dtsi"
+#include "rv1126-evb-v12.dtsi"
+#include "rv1126-evb-v10-robot.dtsi"
 
 / {
 	model = "Rockchip RV1126 EVB DDR3 Board For Robot";
diff --git a/arch/arm/boot/dts/rv1126-evb-ddr3-v12-spi-nor.dts b/arch/arm/boot/dts/rv1126-evb-ddr3-v12-spi-nor.dts
index 38a64e667a1d..ffd9319c7a38 100644
--- a/arch/arm/boot/dts/rv1126-evb-ddr3-v12-spi-nor.dts
+++ b/arch/arm/boot/dts/rv1126-evb-ddr3-v12-spi-nor.dts
@@ -5,7 +5,7 @@
 
 /dts-v1/;
 #include "rv1126.dtsi"
-#include "rv11xx-evb-v12.dtsi"
+#include "rv1126-evb-v12.dtsi"
 
 / {
 	model = "Rockchip RV1126 SPI NOR DDR3 Board";
diff --git a/arch/arm/boot/dts/rv1126-evb-ddr3-v12-tb-emmc.dts b/arch/arm/boot/dts/rv1126-evb-ddr3-v12-tb-emmc.dts
index cede239b95e9..1fc512ce3166 100644
--- a/arch/arm/boot/dts/rv1126-evb-ddr3-v12-tb-emmc.dts
+++ b/arch/arm/boot/dts/rv1126-evb-ddr3-v12-tb-emmc.dts
@@ -5,9 +5,9 @@
 
 /dts-v1/;
 #include "rv1126.dtsi"
-#include "rv11xx-evb-v12.dtsi"
-#include "rv11xx-thunder-boot-emmc.dtsi"
-#include "rv11xx-evb-thunder-boot.dtsi"
+#include "rv1126-evb-v12.dtsi"
+#include "rv1126-thunder-boot-emmc.dtsi"
+#include "rv1126-evb-thunder-boot.dtsi"
 
 / {
 	model = "Rockchip RV1126 EVB DDR3 V12 with eMMC (Thunder Boot)";
diff --git a/arch/arm/boot/dts/rv1126-evb-ddr3-v12-tb-spi-nor.dts b/arch/arm/boot/dts/rv1126-evb-ddr3-v12-tb-spi-nor.dts
index b3e8c8ebb469..fddc593b63ee 100644
--- a/arch/arm/boot/dts/rv1126-evb-ddr3-v12-tb-spi-nor.dts
+++ b/arch/arm/boot/dts/rv1126-evb-ddr3-v12-tb-spi-nor.dts
@@ -5,9 +5,9 @@
 
 /dts-v1/;
 #include "rv1126.dtsi"
-#include "rv11xx-evb-v12.dtsi"
-#include "rv11xx-thunder-boot-spi-nor.dtsi"
-#include "rv11xx-evb-thunder-boot.dtsi"
+#include "rv1126-evb-v12.dtsi"
+#include "rv1126-thunder-boot-spi-nor.dtsi"
+#include "rv1126-evb-thunder-boot.dtsi"
 
 / {
 	model = "Rockchip RV1126 EVB DDR3 V12 with spi nor (Thunder Boot)";
diff --git a/arch/arm/boot/dts/rv1126-evb-ddr3-v12.dts b/arch/arm/boot/dts/rv1126-evb-ddr3-v12.dts
index 1b74d59eb1d8..69eafa297fa4 100644
--- a/arch/arm/boot/dts/rv1126-evb-ddr3-v12.dts
+++ b/arch/arm/boot/dts/rv1126-evb-ddr3-v12.dts
@@ -4,7 +4,7 @@
  */
 /dts-v1/;
 #include "rv1126.dtsi"
-#include "rv11xx-evb-v12.dtsi"
+#include "rv1126-evb-v12.dtsi"
 / {
 	model = "Rockchip RV1126 EVB DDR3 V12 Board";
 	compatible = "rockchip,rv1126-evb-ddr3-v12", "rockchip,rv1126";
diff --git a/arch/arm/boot/dts/rv1126-evb-ddr3-v13-robot.dts b/arch/arm/boot/dts/rv1126-evb-ddr3-v13-robot.dts
index 1f3ebe352436..749d11b1c54c 100644
--- a/arch/arm/boot/dts/rv1126-evb-ddr3-v13-robot.dts
+++ b/arch/arm/boot/dts/rv1126-evb-ddr3-v13-robot.dts
@@ -4,8 +4,8 @@
  */
 /dts-v1/;
 #include "rv1126.dtsi"
-#include "rv11xx-evb-v13.dtsi"
-#include "rv11xx-evb-v10-robot.dtsi"
+#include "rv1126-evb-v13.dtsi"
+#include "rv1126-evb-v10-robot.dtsi"
 
 / {
 	model = "Rockchip RV1126 EVB DDR3 V13 Board For Robot";
diff --git a/arch/arm/boot/dts/rv1126-evb-ddr3-v13-tb-emmc.dts b/arch/arm/boot/dts/rv1126-evb-ddr3-v13-tb-emmc.dts
index 53f5e8d62ddf..a9aad6045f01 100644
--- a/arch/arm/boot/dts/rv1126-evb-ddr3-v13-tb-emmc.dts
+++ b/arch/arm/boot/dts/rv1126-evb-ddr3-v13-tb-emmc.dts
@@ -5,9 +5,9 @@
 
 /dts-v1/;
 #include "rv1126.dtsi"
-#include "rv11xx-evb-v13.dtsi"
-#include "rv11xx-thunder-boot-emmc.dtsi"
-#include "rv11xx-evb-thunder-boot.dtsi"
+#include "rv1126-evb-v13.dtsi"
+#include "rv1126-thunder-boot-emmc.dtsi"
+#include "rv1126-evb-thunder-boot.dtsi"
 
 / {
 	model = "Rockchip RV1126 EVB DDR3 V13 with eMMC (Thunder Boot)";
diff --git a/arch/arm/boot/dts/rv1126-evb-ddr3-v13.dts b/arch/arm/boot/dts/rv1126-evb-ddr3-v13.dts
index 5dd95a841ae3..69d92934d994 100644
--- a/arch/arm/boot/dts/rv1126-evb-ddr3-v13.dts
+++ b/arch/arm/boot/dts/rv1126-evb-ddr3-v13.dts
@@ -4,7 +4,7 @@
  */
 /dts-v1/;
 #include "rv1126.dtsi"
-#include "rv11xx-evb-v13.dtsi"
+#include "rv1126-evb-v13.dtsi"
 / {
 	model = "Rockchip RV1126 EVB DDR3 V13 Board";
 	compatible = "rockchip,rv1126-evb-ddr3-v13", "rockchip,rv1126";
diff --git a/arch/arm/boot/dts/rv11xx-evb-thunder-boot.dtsi b/arch/arm/boot/dts/rv1126-evb-thunder-boot.dtsi
similarity index 100%
rename from arch/arm/boot/dts/rv11xx-evb-thunder-boot.dtsi
rename to arch/arm/boot/dts/rv1126-evb-thunder-boot.dtsi
diff --git a/arch/arm/boot/dts/rv11xx-evb-v10-robot.dtsi b/arch/arm/boot/dts/rv1126-evb-v10-robot.dtsi
similarity index 100%
rename from arch/arm/boot/dts/rv11xx-evb-v10-robot.dtsi
rename to arch/arm/boot/dts/rv1126-evb-v10-robot.dtsi
diff --git a/arch/arm/boot/dts/rv11xx-evb-v10.dtsi b/arch/arm/boot/dts/rv1126-evb-v10.dtsi
similarity index 100%
rename from arch/arm/boot/dts/rv11xx-evb-v10.dtsi
rename to arch/arm/boot/dts/rv1126-evb-v10.dtsi
diff --git a/arch/arm/boot/dts/rv11xx-evb-v12.dtsi b/arch/arm/boot/dts/rv1126-evb-v12.dtsi
similarity index 97%
rename from arch/arm/boot/dts/rv11xx-evb-v12.dtsi
rename to arch/arm/boot/dts/rv1126-evb-v12.dtsi
index 521cb8fa21a4..6014ba7b7550 100644
--- a/arch/arm/boot/dts/rv11xx-evb-v12.dtsi
+++ b/arch/arm/boot/dts/rv1126-evb-v12.dtsi
@@ -3,7 +3,7 @@
  * Copyright (c) 2020 Rockchip Electronics Co., Ltd.
  */
 
-#include "rv11xx-evb-v10.dtsi"
+#include "rv1126-evb-v10.dtsi"
 / {
 	/delete-node/ vdd-npu;
 	/delete-node/ vdd-vepu;
diff --git a/arch/arm/boot/dts/rv11xx-evb-v13.dtsi b/arch/arm/boot/dts/rv1126-evb-v13.dtsi
similarity index 89%
rename from arch/arm/boot/dts/rv11xx-evb-v13.dtsi
rename to arch/arm/boot/dts/rv1126-evb-v13.dtsi
index d4d361b92ea9..1872b3bce114 100644
--- a/arch/arm/boot/dts/rv11xx-evb-v13.dtsi
+++ b/arch/arm/boot/dts/rv1126-evb-v13.dtsi
@@ -3,7 +3,7 @@
  * Copyright (c) 2020 Rockchip Electronics Co., Ltd.
  */
 
-#include "rv11xx-evb-v12.dtsi"
+#include "rv1126-evb-v12.dtsi"
 
 &backlight {
 	pwms = <&pwm0 0 25000 0>;
diff --git a/arch/arm/boot/dts/rv11xx-ipc.dtsi b/arch/arm/boot/dts/rv1126-ipc.dtsi
similarity index 100%
rename from arch/arm/boot/dts/rv11xx-ipc.dtsi
rename to arch/arm/boot/dts/rv1126-ipc.dtsi
diff --git a/arch/arm/boot/dts/rv1126-ipc2-ddr3-v10.dts b/arch/arm/boot/dts/rv1126-ipc2-ddr3-v10.dts
index 864985c6c324..2ca44d47e1f8 100644
--- a/arch/arm/boot/dts/rv1126-ipc2-ddr3-v10.dts
+++ b/arch/arm/boot/dts/rv1126-ipc2-ddr3-v10.dts
@@ -5,7 +5,7 @@
 
 /dts-v1/;
 #include "rv1126.dtsi"
-#include "rv11xx-ipc.dtsi"
+#include "rv1126-ipc.dtsi"
 #include <dt-bindings/input/input.h>
 
 / {
diff --git a/arch/arm/boot/dts/rv1126-sphericalipc-ddr3-v10.dts b/arch/arm/boot/dts/rv1126-sphericalipc-ddr3-v10.dts
index 76ebeec14447..c572b42ab5a5 100644
--- a/arch/arm/boot/dts/rv1126-sphericalipc-ddr3-v10.dts
+++ b/arch/arm/boot/dts/rv1126-sphericalipc-ddr3-v10.dts
@@ -5,7 +5,7 @@
 
 /dts-v1/;
 #include "rv1126.dtsi"
-#include "rv11xx-ipc.dtsi"
+#include "rv1126-ipc.dtsi"
 #include <dt-bindings/input/input.h>
 
 / {
diff --git a/arch/arm/boot/dts/rv11xx-thunder-boot-emmc.dtsi b/arch/arm/boot/dts/rv1126-thunder-boot-emmc.dtsi
similarity index 94%
rename from arch/arm/boot/dts/rv11xx-thunder-boot-emmc.dtsi
rename to arch/arm/boot/dts/rv1126-thunder-boot-emmc.dtsi
index 03680f066f4a..08b87382a8c7 100644
--- a/arch/arm/boot/dts/rv11xx-thunder-boot-emmc.dtsi
+++ b/arch/arm/boot/dts/rv1126-thunder-boot-emmc.dtsi
@@ -3,7 +3,7 @@
  * Copyright (c) 2020 Rockchip Electronics Co., Ltd.
  */
 
-#include "rv11xx-thunder-boot.dtsi"
+#include "rv1126-thunder-boot.dtsi"
 
 / {
 	reserved-memory {
diff --git a/arch/arm/boot/dts/rv11xx-thunder-boot-spi-nor.dtsi b/arch/arm/boot/dts/rv1126-thunder-boot-spi-nor.dtsi
similarity index 89%
rename from arch/arm/boot/dts/rv11xx-thunder-boot-spi-nor.dtsi
rename to arch/arm/boot/dts/rv1126-thunder-boot-spi-nor.dtsi
index 64b2715b6d20..fb8ce5530e13 100644
--- a/arch/arm/boot/dts/rv11xx-thunder-boot-spi-nor.dtsi
+++ b/arch/arm/boot/dts/rv1126-thunder-boot-spi-nor.dtsi
@@ -3,7 +3,7 @@
  * Copyright (c) 2020 Rockchip Electronics Co., Ltd.
  */
 
-#include "rv11xx-thunder-boot.dtsi"
+#include "rv1126-thunder-boot.dtsi"
 
 / {
 	thunder_boot_spi_nor: thunder-boot-spi-nor {
diff --git a/arch/arm/boot/dts/rv11xx-thunder-boot.dtsi b/arch/arm/boot/dts/rv1126-thunder-boot.dtsi
similarity index 100%
rename from arch/arm/boot/dts/rv11xx-thunder-boot.dtsi
rename to arch/arm/boot/dts/rv1126-thunder-boot.dtsi
-- 
2.35.3

